src/components/Overlay.js

```diff
@@ -17,6 +17,26 @@ import LIFECYCLE from '../constants/lifecycle';
 
 import Spotlight from './Spotlight';
 
+(() => {
```

Comment:
Why do you need this iife?

Reply:
you can create a reusable function in modules/dom and initialize it when the component has mounted and canUseDOM